internal/config/app.go

```diff
@@ -86,6 +86,7 @@ func (c *Config) App(n string, ctx *hcl.EvalContext) (*App, error) {
 	// Build a new context with our app-scoped values
 	ctx = ctx.NewChild()
 	addPathValue(ctx, pathData)
+	addAppValue(ctx, rawApp.Name)
```

Comment:
I'd just call `addMapVariable` directly since `addAppValue` isn't reused (like path value is).